### PR TITLE
pr transfer: prettyRainfallPlot update

### PR DESCRIPTION
**Pull request by Kdreval**
Krysta did all the heavy lifting. I just migrated the update from parent repo.
This PR solves issue #16 
Here is the generated plot in example
image

### CODE REVIEW

File: R/prettyRainfallPlot.R [hunk 1]
```diff
@@ -1,353 +1,556 @@
...
-#' @param label_sv Boolean argument to specify whether label SVs or not. Only supported if a specific chromosome or zoom in region are specified.
-#' @param this_seq_type Specify one of "genome" or "capture" when relying on the function to obtain mutations from a region (i.e. if you haven't provided a MAF or single sample_id)
-#' @param plot_title Specify the title for the returned plot, default is "my_plot".
+#' @param maf_path Specify path to MAF file if it is not already loaded into
+#'      data frame.
+#' @param zoom_in_region Provide a specific region in the format
+#'      "chromosome:start-end" to zoom in to a specific region.
+#' @param label_sv Boolean argument to specify whether label SVs or not with
+#'      green line on rainfall plot.
+#' @param annotate_sv Boolean argument to specify whether to restrict SVs to
```
Comment (mattssca) [inline R/prettyRainfallPlot.R:24]: This parameter seems to be duplicated in the function documentation.

File: R/prettyRainfallPlot.R [hunk 2]
```diff
@@ -1,353 +1,556 @@
...
+#'      those annotated with the annotate_sv function (i.e. relevant oncogenes).
+#' @param this_seq_type Specify one of "genome" or "capture" when relying on the
+#'      function to obtain mutations from a region (i.e. if you haven't provided
+#'      a MAF or single sample_id)
+#' @param plot_title Specify the title for the returned plot, default is
+#'      "my_plot".
+#' @param annotate_sv Optionally annotate intrachromosomal SVs to label the gene
```
Comment (mattssca) [inline R/prettyRainfallPlot.R:31]: Here it is again...

**Review by mattssca — COMMENTED** [review]
I think the way these updates were added to this function makes the review rather difficult. It's not straight-forward to see exactly what changes are introduced. I think this is because The initial PR (on GAMBLR main repo), was updating a different version of this plotting function. i.e. this function has been overhauled after its migration to GAMBLR.viz. 

However, is the goal for this function to now take unannotated SVs (raw) as requested in issue #16? If so, is there an example to try this functionality? Assuming this is what the new Boolean parameter is controlling (`annotate_sv`)?

Do we also want to improve this function to take a user-provided SV data frame? I.e. bypassing the internal calls of GAMBLR functions to return SVs? Perhaps this feature is intended as a future improvement. 

I will continue to work with this updated code and test different use-cases. But still wanted to provide some initial feedback that perhaps, would make the review process a bit more streamlined.

**Comment by Kdreval** [comment]
Sorry for cloberring this, the original PR to see the exact changes compared to master is [here](https://github.com/morinlab/GAMBLR/pull/251/files#diff-2ee57637b77c38eb625492f80e8c6d88b08103cce85eb89198a2e7f2f3b1832f)

**Comment by Kdreval** [comment]
Thanks @mattssca !
I have added more examples, and added feature of handling user-provided SVs. This is ready for re-review now!

**Comment by mattssca** [comment]
Thanks, I will have a look at this alter today!

**Review by mattssca — APPROVED** [review]
Thanks!

**Comment by Kdreval** [comment]
Thanks Adam! While working on troubleshooting Vlad's thing I also found something to cleanup here and will soon push one more update here - just waiting for the devtools check to complete

**Comment by mattssca** [comment]
Sounds good!

**Pull request merged**